packages/@lwc/template-compiler/src/codegen/codegen.ts

```diff
@@ -356,6 +358,10 @@ export default class CodeGen {
         );
     }
 
+    getScopedSlotFactory(callback: t.FunctionExpression, slotName: t.SimpleLiteral) {
```

Comment:
minor nit: function should be next to `getSlot` function for easier conceptual organization.